GoogleDataTransport/GDTWatchOSTestApp/GDTWatchOSTestAppWatchKitExtension/InterfaceController.swift

```diff
@@ -21,7 +21,8 @@ import Dispatch
 import GoogleDataTransport
 
 class InterfaceController: WKInterfaceController {
-  let transport: GDTCORTransport = GDTCORTransport(mappingID: "1234", transformers: nil, target: GDTCORTarget.test.rawValue)!
+  let transport: GDTCORTransport = GDTCORTransport(mappingID: "1234", transformers: nil,
+                                                   target: GDTCORTarget.test.rawValue)!
```

Comment:
If you choose test, no network request will be sent. Is that intentional?

Reply:
This test app is in GDT not GDTCCT, I think it is intentional not to sent any request.